com.mudita.mudita-center.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.mudita.mudita-center
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: run.sh
+separate-locales: false
+rename-desktop-file: muditacenter.desktop
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=all
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.PowerManagement
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.gnome.SettingsDaemon
+  - --system-talk-name=org.freedesktop.login1
+modules:
+  - name: muditacenter
+    buildsystem: simple
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/mudita/mudita-center/releases/download/2.5.0/Mudita-Center.AppImage
+        sha256: 0fbef5c2020a8f9574d438741c5907f00a828c48927b48f8b12817881a2176a5
+      - type: file
+        path: run.sh
+      - type: file
+        path: muditacenter.desktop
+    build-commands:
+      - chmod +x Mudita-Center.AppImage
+      - ./Mudita-Center.AppImage --appimage-extract
+      - |
+        for size in 16 32 48 64 128 256 512; do
+          install -Dm0644 "squashfs-root/usr/share/icons/hicolor/${size}x${size}/apps/Mudita Center.png" "${FLATPAK_DEST}/share/icons/hicolor/${size}x${size}/apps/com.mudita.mudita-center.png"
```

Comment:
you should use `rename-icon` instead (it's in the top level of the manifest.

Reply:
How would I do this for all the icons? Would I still have to `install` them into their destination as their original name before using `rename-icon`?

Reply:
`/app/usr` make no sense (it's a bug). so if you address then you are golden.

now if you built from sources...